src/noisepy/seis/datatypes.py

```diff
@@ -239,6 +242,24 @@ def get_storage_options(self, path: str) -> Dict[str, Any]:
     def dt(self) -> float:
         return 1.0 / self.samp_freq
 
+    def load_stations(self, stations_list=None) -> List[str]:
+        if stations_list is None:
+            # Load the list from the file
+            with open(self.stations_file, "r") as file:
+                self.stations = file.read().splitlines()
+        else:
+            self.stations = stations_list
+        return 0
+
+    def save_stations(self, value: List[str]):
+        if self.stations_file:
+            # Save the list to the file
+            with open(self.stations_file, "w") as file:
+                file.write("\n".join(value) + "\n")
+            # Set stations field to Empty List
+            self.stations = []
```

Comment:
Why? It may be confusing that saving the list of stations empties this field

Reply:
As mentioned in the 3rd point of the 'Read station list from a file ' issue - "When saving, save the list to the file and set the stations field to none", I am setting the stations field to None. Even I was not sure if it is correct or not but I followed the instructions mentioned. I could remove this line of code if not needed.